ingestion/src/metadata/ingestion/source/dashboard/tableau/metadata.py

```diff
@@ -375,13 +372,14 @@ def yield_dashboard_chart(
                     self.status.filter(chart.name, "Chart Pattern not allowed")
                     continue
                 site_url = (
-                    f"site/{self.service_connection.siteUrl}/"
+                    f"/site/{self.service_connection.siteUrl}/"
                     if self.service_connection.siteUrl
                     else ""
                 )
                 workbook_chart_name = ChartUrl(chart.contentUrl)
 
                 chart_url = (
+                    f"{str(self.service_connection.hostPort)}/"
```

Comment:
let's use the clean uri method here you can refer [metabase](https://github.com/open-metadata/OpenMetadata/blob/488a03df3bae73d498be7743663a7b4cf47ac109/ingestion/src/metadata/ingestion/source/dashboard/metabase/metadata.py#L96) 

basically the idea is if user already put uri like `http://tableau.net/` then this will fail as an additional `/` will be appended here